eul/AppDelegate.swift

```diff
@@ -41,13 +41,13 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSWindowDelegate {
             styleMask: [.titled, .closable, .miniaturizable, .resizable, .fullSizeContentView],
             backing: .buffered, defer: false
         )
+        window.title = "ui.preferences".localized()
         window.center()
         window.setFrameAutosaveName("Eul Preferences")
         window.contentView = NSHostingView(rootView: contentView)
         window.isReleasedWhenClosed = false
         window.standardWindowButton(.zoomButton)?.isHidden = true
         window.standardWindowButton(.miniaturizeButton)?.isHidden = true
-        window.makeKeyAndOrderFront(nil)
```

Comment:
will it be better to check if it was launched by clicking the app or at login? it'll be good to show the preference window if user launches the app intentionally